demo/components/demo/components/dropdowns/dropdowns.html

```diff
@@ -0,0 +1,32 @@
+<span class="slds-dropdown-trigger" nglDropdown [(open)]="open" (select)="onSelect($event)">
+  <button class="slds-button slds-button--neutral" nglDropdownTrigger>
+    <span class="slds-truncate">Select an Action</span>
+    <svg aria-hidden="true" class="slds-button__icon slds-button__icon--hint">
+      <use xlink:href="assets/icons/utility-sprite/svg/symbols.svg#down"></use>
+    </svg>
+  </button>
+  <div class="slds-dropdown slds-dropdown--left">
+    <ul class="dropdown__list" role="menu">
+      <li class="slds-dropdown__item" [nglDropdownItem]="'Action1'">
+        <a role="menuitem">
+          <p class="slds-truncate">Action1</p>
+          <svg aria-hidden="true" class="slds-icon slds-icon--x-small slds-icon-text-default slds-m-left--small slds-shrink-none">
```

Comment:
I don't quite follow. `aria` classes are indeed being used for selected items as per the slds guidelines.